src/app/mapmyproperty/esrimap/esrimap.component.html

```diff
@@ -23,6 +23,7 @@
         </div>
         <app-graphic-store #graphicsStore [sketchVM]="sketchVM" 
         [selectedGraphics]="selectedGraphics"
```

Comment:
Needs to address #90 as text selection is not regarded as selected graphic on `esrimap.component.ts`.